cmd/gateway/web_mux_test.go

```
@@ -0,0 +1,67 @@
+		wantBody string
+	}{
+		{"prefixed path routes to bridge", "/gw/01KEXAMPLEGATEWAYID00000000/terminal", http.StatusOK, "bridge"},
+		{"unprefixed path still routes to bridge", "/terminal", http.StatusOK, "bridge"},
+		{"health is plain OK", "/health", http.StatusOK, "ok"},
+		{"unknown path is 404", "/nope", http.StatusNotFound, "404 page not found\n"},
+		{"wrong gateway id prefix is 404", "/gw/01KOTHERID/terminal", http.StatusNotFound, "404 page not found\n"},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			req := httptest.NewRequest(http.MethodGet, tt.path, nil)
```

golangci-lint (2.11.4)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)